ui/widget.js

```diff
@@ -258,6 +259,7 @@ $.Widget.prototype = {
 		this.bindings = $();
 		this.hoverable = $();
 		this.focusable = $();
+		this.classObject = {};
```

Comment:
Im open to other names, Im not sure managed classes as that implies its a list of classes not a collection of elements ( not that my name is any more clear ) @scottgonzalez any thoughts on a better name?  `classesElements` , `managedElements` ? `classesElementLookup` .... I don't know